pkg/finder/fsfinder.go

```diff
@@ -135,10 +158,31 @@ func (fsf FileSystemFinder) findOne(pathRoot string) ([]FileMetadata, error) {
 
 				walkFileExtension := strings.TrimPrefix(filepath.Ext(path), ".")
 
+				// Checking for file name matching with additional files
+				// Check matching base name, return vals with passed KVPair value as fileType
+				//if _, ok := fsf.AdditionalFiles[dirEntry.Name()]; ok {
+				//	extensionLowerCase := AdditionalFiles[dirEntry.Name()]
+				//}
+
+				if len(walkFileExtension) == 0 {
+					var ok bool
+					walkFileExtension, ok = fsf.AdditionalFiles[dirEntry.Name()]
+					if !ok {
+						walkFileExtension = ""
+					}
+				}
+
+				fmt.Println(path)
+				fmt.Println(walkFileExtension)
+
+				// Checking for case sensitive exclusion
 				if _, ok := fsf.ExcludeFileTypes[walkFileExtension]; ok {
+					fmt.Println("excluded")
```

Comment:
Reminder to remove before MR is finalized
```suggestion
					fmt.Println("excluded")
```